Dockerfile.new

```diff
@@ -0,0 +1,11 @@
+FROM registry.redhat.io/ubi8/python-36
+USER 0
+COPY Pipfile Pipfile
+COPY Pipfile.lock Pipfile.lock
+COPY run_server.sh run_server.sh
+RUN yum install -y git gcc python3-devel && pip3 install pipenv pip pipenv-to-requirements && pip3 install -U pip && pipenv run pipenv_to_requirements -f \
+    && pip3 install -r requirements.txt && yum remove -y nodejs gcc python3-devel atlas && yum clean all
+COPY rbac rbac
+COPY scripts scripts
+COPY openshift/s2i/bin/run run
+ENTRYPOINT ["./run"]
```

Comment:
Will this Dockerfile be expected to be used for the app service as well as the scheduler/worker? Asking because we can't run this entrypoint for the scheduler/worker, those have different startup commands: https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/rbac/worker.yaml#L49-L54

Sorry if I'm missing something. 😄

Reply:
I do see you have those in the clowder files below, but they should be run off the base image without the ./run script being run, I believe.

Reply:
I think the `command` overrides the entrypoint, but yes, we can change this so that the single image has no `command/entrypoint` per se, and we use the command in ClowderApp instead to invoke. Would you be OK with that?

Reply:
Seems like that should work! That way we can reuse the one image for all the services, which is the goal?